packages/extension-bubble-menu/package.json

```diff
@@ -29,12 +29,16 @@
     "dist"
   ],
   "peerDependencies": {
-    "@tiptap/core": "^2.0.0-beta.193"
+    "@tiptap/core": "^2.0.0-beta.193",
+    "prosemirror-state": "^1.4.1",
+    "prosemirror-view": "^1.28.2"
   },
   "dependencies": {
+    "tippy.js": "^6.3.7",
+    "lodash": "^4.17.21",
+    "@tiptap/core": "^2.0.0-beta.193",
     "prosemirror-state": "^1.4.1",
-    "prosemirror-view": "^1.28.2",
-    "tippy.js": "^6.3.7"
+    "prosemirror-view": "^1.28.2"
```

Comment:
I anticipate that these three remaining tiptap & prosemirror `dependencies` will still cause trouble. I think if you moved them to `devDependencies` then things should work more smoothly.